### ISSUE TITLE
bean-report and custom directives

### ISSUE THREAD

**Issue by blais** [beancount/beancount#129]
**[Original report](https://bitbucket.org/blais/beancount/issue/129) by Bruno Vernier (Bitbucket: [brunov25](https://bitbucket.org/brunov25), GitHub: [brunov25](https://github.com/brunov25)).**

----------------------------------------

bean-report beanfile [h]ledger barfs the following when fed custom directives:

"Traceback (most recent call last):
  File "/usr/local/bin/bean-report", line 2, in <module>
    from beancount.reports.report import main; main()
  File "/usr/local/lib/python3.4/dist-packages/beancount/reports/report.py", line 250, in main
    chosen_report.render(entries, errors, options_map, args.format, outfile)
  File "/usr/local/lib/python3.4/dist-packages/beancount/reports/base.py", line 113, in render
    result = render_method(entries, errors, options_map, outfile)
  File "/usr/local/lib/python3.4/dist-packages/beancount/reports/convert_reports.py", line 139, in render_ledger
    file.write(ledger_printer(entry))
  File "/usr/local/lib/python3.4/dist-packages/beancount/reports/convert_reports.py", line 155, in __call__
    method = getattr(self, obj.__class__.__name__)
AttributeError: 'LedgerPrinter' object has no attribute 'Custom'

**Comment by blais** [comment]
**Original comment by Bruno Vernier (Bitbucket: [brunov25](https://bitbucket.org/brunov25), GitHub: [brunov25](https://github.com/brunov25)).**

----------------------------------------

to be clear, all other bean-report keywords work with custom directives except for hledger and ledger

**Comment by blais** [comment]
**Original comment by Saša Janiška (Bitbucket: [gour](https://bitbucket.org/gour), GitHub: [gour](https://github.com/gour)).**

----------------------------------------

Just to confirm this one which I've encountered after adding:


```
2015-01-01 custom "budget" Expenses:Food             "yearly"         someamount HRK
```

to my beancount/ledger file.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #129: Ignore custom directives for Ledger and HLedger conversion.
